scientist-softserv/harvard-curiosity
# Delete rtl_show_presenter.rb

**Delete rtl_show_presenter.rb**
* * *


# What does this Pull Request do?
It deletes rtl_show_presenter.rb -- file was causing issues in prod and appears we don't need it for function?

# How should this be tested?

Run locally in prod mode if possible (also on dev right now). Confirm site works and the links for facets still work.

# Test coverage
Yes/No: Are changes in this pull-request covered by:
- unit tests?
- integration tests?

# Interested parties
Tag (@ mention) interested parties
